python/cuml/holtwinters/holtwinters.pyx

```diff
@@ -0,0 +1,199 @@
+                                  <float*> &alpha_f[0],
+                                  <float*> &beta_f[0],
+                                  <float*> &gamma_f[0],
+                                  <float*> &SSE_error_f[0],
+                                  <float*> &forecast_f[0])
+
+            self.alpha = alpha_f
+            self.beta = beta_f
+            self.gamma = gamma_f
+            self.SSE_error = SSE_error_f
+            self.forecasted_points = forecast_f
+
+        elif self.dtype == np.float64:
+            alpha_d = np.ascontiguousarray(np.empty(self.batch_size,
+                                                    dtype=self.dtype))
+            beta_d = np.ascontiguousarray(np.empty(self.batch_size,
+                                                   dtype=self.dtype))
+            gamma_d = np.ascontiguousarray(np.empty(self.batch_size,
+                                                    dtype=self.dtype))
+            SSE_error_d = np.ascontiguousarray(np.empty(self.batch_size,
+                                                        dtype=self.dtype))
+            forecast_d = np.ascontiguousarray(np.empty(self.batch_size*self.h,
+                                                       dtype=self.dtype))
+
+            HoltWintersFitPredict(<int> n_rows, <int> self.batch_size,
+                                  <int> self.frequency, <int> self.h,
+                                  <int > self.start_periods,
+                                  <SeasonalType> self._cpp_stype,
+                                  <double*> input_ptr,
+                                  <double*> &alpha_d[0],
+                                  <double*> &beta_d[0],
+                                  <double*> &gamma_d[0],
+                                  <double*> &SSE_error_d[0],
+                                  <double*> &forecast_d[0])
+
+            self.alpha = alpha_d
+            self.beta = beta_d
+            self.gamma = gamma_d
+            self.SSE_error = SSE_error_d
+            self.forecasted_points = forecast_d
+
+        else:
+            raise TypeError("HoltWinters supports only float32"
+                            " and float64 input, but input type "
+                            + str(self.dtype) + " passed.")
+
+        self.fit_executed_flag = True
+        del(X_m)
+
+    def score(self, index):
+
+        index = index - 1
+        if self.fit_executed_flag:
+            return self.SSE_error[index]
+        else:
+            raise Exception("Fit() the model before score()")
+
+    def predict(self, n, h):
+        if h > 50:
+            raise Exception("Default value of forecasted points is 50."
+                            " To get more points, execute fit() function"
+                            " with pointsToForecast > 50. \nUsage : fit"
+                            "(numpyInputList, pointsToForecast) \n ")
+
+        if self.fit_executed_flag:
+            forecast = []
+            n = n-1
```

Comment:
Please check if the value of `n` is within bounds or not and if not please throw an error message describing the limit. This will make it easier for the user to debug.